lib/galaxy/webapps/galaxy/controllers/visualization.py

```diff
@@ -1015,6 +1017,45 @@ def phyloviz( self, trans, id=None, dataset_id=None, tree_index=0, **kwargs ):
             }
         return trans.fill_template_mako( "visualization/phyloviz.mako", data=data, config=config )
 
+    @web.expose
+    @web.require_login( "run Galaxy Interactive Environments" )
```

Comment:
I believe this shows an error message that says "You must login to ... run GIEs".

It's not an API, it's an older mako template. I would have liked to write a new fancy JS interface, but I did not know how to do that in the context of the other JS-only interfaces that were available.